public/app/features/variables/adhoc/adapter.ts

```diff
@@ -8,6 +8,9 @@ import { adHocVariableReducer, initialAdHocVariableModelState } from './reducer'
 import { AdHocVariableEditor } from './AdHocVariableEditor';
 import { setFiltersFromUrl } from './actions';
 import * as urlParser from './urlParser';
+import { isAdHoc, isLegacyAdHocDataSource } from '../guard';
+import { getDataSourceSrv } from '../../../../../packages/grafana-runtime';
+import { getDataSourceRef } from '../../../../../packages/grafana-data';
```

Comment:
from packages? not relative paths

Reply:
Thanks for catching that, since the yarn pnp PR my IDE suggests these paths. Need to figure that out.